src/server/routes/signIn.ts

```diff
@@ -603,6 +604,24 @@ router.get(
 			);
 		}
 
+		// GOOGLE ONE TAP FLOW
+		// If the provider is `google` and we have a `signInEmail` parameter, assume
+		// that we're going down the Google One Tap sign in route. We cannot use the IDX API in
+		// this route due to a limitation where we can't(easily) set parameters for the Google IdP
+		// authentication URL as it is generated by Okta.
+		if (signInEmail && socialIdp === 'google') {
+			return await performAuthorizationCodeFlow(req, res, {
+				idp,
+				closeExistingSession: true,
+				scopes: scopesForAuthentication,
+				redirectUri: ProfileOpenIdClientRedirectUris.AUTHENTICATION,
+				loginHint: signInEmail,
+				extraData: {
+					socialProvider: socialIdp,
+				},
+			});
+		}
```

Comment:
this is in `SignIn.ts` but it would also result in an account created if the user doesn't already have one, right ?